website/views/options/social/tavern.jade

```diff
@@ -34,7 +34,7 @@
                 a(target='_blank', href='/static/community-guidelines')=env.t('communityGuidelines')
             tr
               td
-                a(target='_blank', href='http://habitrpg.wikia.com/wiki/Board:The_Archery_-_LFG')=env.t('lfgPosts')
+                a(target='_blank', href='https://habitrpg.com/#/options/groups/guilds/f2db2a7f-13c5-454d-b3ee-ea1f5089e601')=env.t('lfgPosts')
```

Comment:
Let's get rid of the `target='_blank'` since the link is opening in site, we don't need it to open in a separate tab.